src/libs/OptionsListUtils.js

```diff
@@ -119,72 +119,45 @@ function createOption(personalDetailList, report) {
 }
 
 /**
- * Rebuild the options. We are throttling this so the options are only rebuilt at most
- * once per second. It can be expensive to rebuild options so we slow this down a bit.
+ * Rebuild the options
+ * @param {Boolean} sortByLastMessageTimestamp
  */
-const rebuildOptions = _.throttle(() => {
+function rebuildOptions(sortByLastMessageTimestamp = false) {
```

Comment:
The reason for adding this default param is because some of the methods we use for testing below, like `getSearchOptions`, should sort by `lastVisitedTimestamp` and not the `lastMesssageTimestamp`? Is this to fit certain testing cases?

Reply:
Nope we are adding behavior because this was specced out wrong..

Reply:
Basically, we only just realized the LHN is supposed to be sorted by the last message not last visited.

Reply:
Ah ok, so the SideBar ordering should change when a new message is sent, not when it's clicked on

Reply:
Is it possible to add tests for this change? Or is that out of the scope of the PR?

Reply:
I can we just have to add timestamp data to the reports should be easy.